src/codeFlowLogin.ts

```diff
@@ -231,7 +231,7 @@ function createTerminateServer(server: http.Server) {
 		});
 	});
 	return async () => {
-		const result = new Promise<void>(resolve => server.close(resolve));
+		const result = new Promise<void>(() => server.close());
 		for (const id in sockets) {
```

Comment:
Do we have test cases for this extension and if so where are they (I didn't see them in the OneNote)? With all these engineering changes lately, we need to make sure we have some regular testing to reduce the chance of regressions

Reply:
Please make sure the tests cover both methods of signing in ("sign in" and "sign in with device code"). And make sure that CTI is running these as a part of their regular testing

Reply:
Will do 👍